README.md

```diff
@@ -51,6 +51,16 @@ The `data` property is a Typed Array containing the pixel data. It is a `Uint8Ar
 * `yResolution`
 * `resolutionUnit`
 
+### tiff.countPages(data)
+
+Returns the number of IFDs (pages) in the file.
+
+### tiff.checkMultiPage(data)
```

Comment:
Maybe we can call this `hasMultiPage`?

Reply:
Sweet! Agree, `checkMultiPage` is not awesome. I was hesitant to use `isX` and `hasX` style naming if it would convey it's a boolean variable, instead of a function (verb) that returns a boolean.

In the class I use a getter so there it makes perfect sense to name them `isMultiPage` and `pageCount`. To keep it simple, what about naming the exported functions the same as the class props (eg exporting `tiff.isMultiPage()` and `tiff.pageCount()`?